backend/src/routes/admin/email/get_templates.rs

```diff
@@ -0,0 +1,19 @@
+use crate::services::emails::EmailTemplate;
+use crate::utils::error::Error;
+use actix_web::{HttpResponse, get};
+
+#[utoipa::path(
+    responses(
+        (status = 200, description = "Templates successfully retrievied", body = Vec<EmailTemplate>),
+        (status = 500, description = "Internal server error.")
+    ),
+    security(
+        ("access_token" = [])
+    ),
+    tag = "admin/email"
+)]
+#[get("/get_templates")]
```

Comment:
uh, `templates_list`, `templates`, why `get_templates`? I think we don't use names like that for users and other laces